Math.h

```diff
@@ -24,9 +24,9 @@
 #ifndef MATH_H
 #define MATH_H
 
-// Includes.
-#include "Math.enum.h"
-#include "Math.struct.h"
+// Includes
+#include "Math.enum.h"     /*!< enum value */
```

Comment:
Change back to what it was (all 3 lines), include doesn't need documenting. Secondly it's not enum. After the change, it shouldn't show any difference.